server/config.ts

```diff
@@ -85,9 +85,6 @@ export default {
     secret: get('SESSION_SECRET', 'app-insecure-default-session', requiredInProduction),
     expiryMinutes: Number(get('WEB_SESSION_TIMEOUT_IN_MINUTES', 120)),
   },
-  sentry: {
-    dsn: get('SENTRY_DSN', null, notRequiredInProduction),
```

Comment:
We should also remove from `values.yaml`.

Reply:
it's not in there but i was gonna remove it from the k8s secret after deploying

Reply:
sorry, was looking at a stale version of this repository where the `SENTRY_DSN` was still there 👌